### PR TITLE
consistent security group between different envs

### PR DESCRIPTION
**Pull request by brianhu2006**
The different maintain their own security group. It is not easy to maintain rules between differents envs.

### CODE REVIEW

File: playbooks/ci-ceph-swift-rhel/templates/heat_stack.yml [hunk 1]
```diff
@@ -93,97 +93,8 @@ parameters:
 resources:
```
Comment (epwn) [inline playbooks/ci-ceph-swift-rhel/templates/heat_stack.yml]: can you make sure your spacing is consistent across your changes to the templates?

Comment (brianhu2006) [inline playbooks/ci-ceph-swift-rhel/templates/heat_stack.yml]: This field `resources`  is aligned with `parameters`. No space before resources

**Comment by nirajdp76** [comment]
We probably need symlinks in each envs to heat_stack.yml so that our developer workflow is not impacted. 
example: one should still should be able to do deployment using
ursula --ursula-user centos --provisioner=heat --heat-stack-name 'centos_s1' envs/example/ci-full-centos site.yml

**Comment by brianhu2006** [comment]
@nirajdp76 
1. Under the two envs (ci-ceph_swift and ci-full), there is heat_stack.yml.
    /envs/example/ci-ceph_swift
    /envs/example/ci-full

2.if keep placing securitygroup in different envs. it is difficult to maintain security group rules for all envs.

**Comment by nirajdp76** [comment]
My assumption was that heat_stack.yml will be in env/examples/ci** to enable the developers able to use ursula --provisioner=heat.

**Comment by brianhu2006** [comment]
@nirajdp76 
- heat-stack template will be under this directory(playbooks/ci**/template), after heat-stack template generation. All files will be move to envs/example/c**

- After this change ,  all configuration will be kept under this directory envs/example/c**

**Comment by jeffreyzhou** [comment]
test ceph

**Pull request closed without merging**